MIGRATION.md

```diff
@@ -0,0 +1,25 @@
+Breaking changes:
```

Comment:
Why is this in a separate `MIGRATION` file (rather than changelog post merge) 🤔

Reply:
mistakenly committed

Reply:
deleted